### ISSUE TITLE
[@types/react-native] Error when using new accessibilityProps in react-native 0.57: AccessibilityRole, AccessibilityState, accessibilityHint, accessibilityIgnoresInvertColors

### ISSUE THREAD

**Issue by jessamp** [DefinitelyTyped/DefinitelyTyped#29666]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/react-native` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/@types/react-native
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/). 
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @alloy @huhuanming @iRoachie @timwangdev @kamal @nelyousfi @alexdunne @swissmanu @bm-software @tkrotoff @a-tarasyuk @mvdam

In react-native 0.57, they introduced new accessibility props. These have not been added yet to @types, and therefore cannot be used. They will be deprecating some other props in support of these new ones sometime soon. If they can be added that would be great. 

For reference:
https://facebook.github.io/react-native/blog/2018/08/13/react-native-accessibility-updates 
https://facebook.github.io/react-native/docs/accessibility
